### ISSUE TITLE
Adding a filter to block cookie consent popups

### ISSUE THREAD

**Issue by ghost** [Slion/Fulguris#438]
The filters don't block the cookie popups.

**Comment by Slion** [comment]
@Helium314 I don't think we support that right?

**Comment by Helium314** [comment]
I don't know, maybe with the right blocklist it works.
It really depends on the specific filters that handle this, as we don't support cosmetic filters and some AdGuard / uBlock extended syntax.

**Comment by ILeMeNiiz** [comment]
> @Helium314 I don't think we support that right?

Wäre es möglich mehr Listen hinzuzufügen? Wie zb für Cookies? Ich habe es selbst versucht aber ich bin leider kein Experte, es funktionieren nur manche...

Übrigens richtig geiler Browser, heute entdeckt und direkt neuer Standartbrowser

**Comment by Helium314** [comment]
I looked through some filter lists / rules that should handle this, but it looks like all / most use the cosmetic filtering syntax, which is (currently) not supported by Fulguris.

**Comment by Cwpute** [comment]
yeah, i discussed this over with the dev behind Einkbro [(see here)](https://github.com/plateaukao/einkbro/discussions/202) .
Basically, cookie consent needs code injection to work, which is generally beyond the scope of a simple web-browser like Fulguris. Maybe there is another way around ? but i'm not knowledgeable enough on the matter.

**Comment by Cwpute** [comment]
I also asked away on the forum of a popular cookie consent addon [(here)](https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/discussions/3831) what could be done for android web-browsers, but didn't get any answer as of now.

**Comment by Slion** [comment]
Related to #352
